src/js/components/invoice/ConfirmInvoicePage.jsx

```diff
@@ -187,6 +187,9 @@ class ConfirmInvoicePage extends Component {
       if (!_.isNull(startIndex) &&
           this.state.values.invoiceItems.length !== this.state.values.totalCount) {
         this.loadMoreRows({ startIndex: startIndex + this.props.pageSize });
+      } else if (this.state.values.invoiceItems.length === this.state.values.totalCount
```

Comment:
Why are you checking `this.state.values.invoiceItems.length === this.state.values.totalCount`? What does this indicate?

Reply:
When all regular invoice items are loaded then we fetch prepaid items and add to the end of the list of invoice items.

Reply:
Makes sense